com.unity.render-pipelines.universal/Editor/ShaderGUI/BaseShaderGUI.cs

```diff
@@ -315,35 +315,38 @@ public virtual void DrawBaseProperties(Material material)
             }
         }
 
+        private void DrawEmissionTextureProperty()
+        {
+            if ((emissionMapProp == null) || (emissionColorProp == null))
+                return;
+
+            using (new EditorGUI.IndentLevelScope(2))
+            {
+                materialEditor.TexturePropertyWithHDRColor(Styles.emissionMap, emissionMapProp, emissionColorProp, false);
+            }
+        }
+
         protected virtual void DrawEmissionProperties(Material material, bool keyword)
         {
-            var emissive = true;
-            var hadEmissionTexture = emissionMapProp?.textureValue != null;
+            var emissive = materialEditor.EmissionEnabledProperty(); // Emission for GI?
```

Comment:
I think this may lead to different behavior. Previously, if `keyword` was false, `emissive` was always true, meaning the code on line 357 got called. Now it no longer depends on the value of `keyword`. Is it an intended change in behavior?